### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by elenademi**

### CODE REVIEW

**Review by Alzhan-B — APPROVED** [review]

**Review by nikkigraybeal — CHANGES_REQUESTED** [review]
Fantastic job Elena (and Alzahn as well if you worked together)!

Just a couple of things to address before we merge but this is a great start! Feel free to reach out in Slack if you have any questions.

**Review by Alzhan-B — APPROVED** [review]
Elena, looks good! 
Try taking out extra empty lines next time, like 9, 11, 20, 24 and 25. Or you can use Prettier format before committing next time!

File: src/pages/LoginPage/LoginPage.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+function handleSubmit(){
+    console.log ("Hello world!")
+}
+    return(
+   
+    <div className='landingPage'>
+     
+     <h1>Login Page</h1>
+     <form className="formContainer">
+      <label htmlFor="email" >Email</label>
```
Comment (nikkigraybeal) [inline src/pages/LoginPage/LoginPage.js:14]: ah - nice catch on using htmlFor instead of just 'for' here!

**Review by nikkigraybeal — APPROVED** [review]
Nice job implementing the changes, Elena! I think it's ready to merge 🎉 

EDIT: ... with one exception! please pull main into this branch on your local and resolve the merge conflict in App.js. then add, commit and push those changes. Then it should be ready to go 😄

File: src/pages/LoginPage/LoginPage.js (hunk 1 above)
Comment (elenademi) [reply src/pages/LoginPage/LoginPage.js:14]: that was Dasha's idea

**Review by nikkigraybeal — APPROVED** [review]

**Pull request merged**